### ISSUE TITLE
WIN SE80_GIT cmdLineTester_lockWordAlignment_Object_xx failure: Check whether Object contains necessary private fields if any

### ISSUE THREAD

**Issue by JasonFengJ9** [eclipse/openj9#7436]
Failure link
------------

Internal tests.

Optional info
-------------

- intermittent failure (yes|no): 
- regression or new test:  These tests are not new, but just appeared (maybe exclude list or build graph changes)

Failure output (captured from console output)
---------------------------------------------

```
Testing: Check whether Object contains necessary private fields if any
Test start time: 2019/10/10 23:37:28 Eastern Standard Time
Running command: w:\builds\bld_429629/sdk/wa6480\bin\javap -p -bootclasspath w:buildsbld_429629/jvmtest/test/SE80functionalcmdLineTestslockWordAlignmentobject_d.jar java/lang/Object
Time spent starting: 829 milliseconds
Time spent executing: 2969 milliseconds
Test result: FAILED
Output from test:
 [ERR] Error: class not found: java/lang/Object
>> Success condition was not found: [Return code: 0]
>> Required condition was not found: [Output match: \W*private double d;]
>> Required condition was found: [Output match: .*]
>> Required condition was found: [Output match: .*]
```

**Comment by pshipton** [comment]
@llxia

**Comment by llxia** [comment]
This happens only on wins in internal (vmfarm). wins on Jenkins works fine: https://ci.eclipse.org/openj9/view/Test_Functional_Nightly/job/Test_openjdk8_j9_extended.functional_x86-64_windows_Nightly/164

The path is incorrect `w:buildsbld_429629` internally.

**Comment by llxia** [comment]
After fixing the path, the test still failed on wins internally. Need further investigation. I will exclude the test internally for now.

```
Running command: w:\builds\bld_429710/sdk/wi3280\bin\javap -p -bootclasspath w:\builds\bld_429710/jvmtest/test/SE80\functional\cmdLineTests\lockWordAlignment\object_d.jar java/lang/Object
Time spent starting: 156 milliseconds
Time spent executing: 2281 milliseconds
Test result: FAILED
Output from test:
 [ERR] Error: class not found: java/lang/Object
>> Success condition was not found: [Return code: 0]
>> Required condition was not found: [Output match: \W*private double d;]
>> Required condition was found: [Output match: .*]
>> Required condition was found: [Output match: .*]
```

**Comment by llxia** [comment]
The test is fixed by #7530. It should pass on internal windows.
